### DISCUSSION TITLE
New design for 2.0 docs - Need reviews and comments

### DISCUSSION THREAD

**Discussion by umamukkara** [litmuschaos/litmus#2783] [General]
We have a figma design. that has the initial outline, the information experience flow. Need reviews and feedback will be helpful.

https://www.figma.com/proto/MGqgXZ9elrPfRFlWE7AyZ3/Litmus-Docs?page-id=3%3A4&node-id=510%3A0&viewport=532%2C344%2C0.0595678873360157&scaling=min-zoom

cc @prithvi1307 @mallick370

**Comment by mallick370** [comment]
Inviting @ksatchit for Litmus 2.0 Doc design review.

**Comment by mallick370** [comment]
Inviting @ksatchit @divya-mohan0209 @rberrelleza for our new Litmus 2.0 Doc design review. Feel free to comment and also invite others for review.

**Comment by prithvi1307** [comment]
A few comments from my side:

- We can add a tasks/usecases section providing guides and examples of common tasks that one may need to perform when using Litmus.
- Most importantly the Docs should feature and highlight the Contributing section with a Developer Guide (Observability & Platform) to invite more contributors
- We can also document the release process to put it out to the community how we organize each release.
Something like this:
feature freeze, pin dependencies, branch repo, release branch prep, tag release, build/publish, tag next pre-release, verify, promote, etc.

- There can be section talking about APIs. 
- Does the get started section talk about prerequisites or we can have one.
- Airgapped, GitOps, Observability can come into one single section called features.
- Lastly, we can have a Troubleshoot guide which addresses some common issues users have faced till now.

**Comment by rberrelleza** [comment]
I really like the new design, I think it's clear and easy to read! A few comments on the stuff I saw on Figma:

1.  I think a "What is LitmusChaos" section at the beginning would be beneficial since the people visiting the docs might not be experts on Litmus. 
2. I find the order of the "Learn Litmus" section confusing. It might be worth separating it in different personas (e.g. developer that wants to  use litmus for chaos test, vs contributor of litmus, vs admin installing litmus)
3. Also, on the 'learn litmus' section, I feel like there are a lot of sections there. And a few of those are so generic (e.g Learn Litmus, Resources, Security) that it's hard for me to imagine what kind of information they will contain.
3. I'd put the request for star and the link to the GitHub repo as one of the first elements of the doc, as most people might not reach the end of the page.
4. The gif on the right confused me a little bit. It started as a realistic CLI but then it switched into a browser-like UI. I think showing my install, create an experiment and see the result might be enough for a 'first view' kind of thing.

**Comment by ksatchit** [comment]
Inviting @Gbahdeyboh here for context & feedback on the litmus docs structure!!
